canonical-web-and-design/vanilla-framework
# Remove deprecated typography styles

## Done

Removes deprecated styles related to typography

Part of #3744

## QA

- Open [demo](https://vanilla-framework-3956.demos.haus/)
- Make sure Vanilla builds correctly
- Make sure there are no unexpected changes in Percy
